tests/fixtures/expected/doc-reference/index.html

```diff
@@ -4,8 +4,8 @@
     <meta charset="utf-8" />
 </head>
 <body>
-<div class="section" id="a-test-for-doc-link">
-    <h1>A test for doc link<a class="headerlink" href="#a-test-for-doc-link" title="Permalink to this headline">¶</a></h1>
+<div class="section">
+    <h1 id="a-test-for-doc-link">A test for doc link<a class="headerlink" href="#a-test-for-doc-link" title="Permalink to this headline">¶</a></h1>
```

Comment:
@nikophil I believe this is what we want based off of https://github.com/weaverryan/docs-builder/issues/60 correct?

Reply:
this is correct, I think we can leave this PR as is and I'll fix all problems related to headings in the other PR